tox.ini

```
@@ -1,8 +1,9 @@
 [tox]
 minversion = 3.14.6
 envlist =
-    py{38,39,310}-dj{32,41}-wagtail{41,51,52}
-    py311-dj{42}-wagtail{51,52}
+    py{38,39,310}-dj32-wagtail52
+    py{38,39,310,311}-dj42-wagtail{52,60}
+    py{311,312}-dj50-wagtail{52,60}
```

Technically, Python 3.10 is also supported by Django 5.0. https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support

```suggestion
    py{310,311,312}-dj50-wagtail{52,60}
```

---

Thanks @Stormheg , applied this one!